src/components/tag/Tag.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { DEFAULT_TEXT_SIZE } from '../text/Text.constants';
+
+import PTag from './Tag.props';
+import TagRoot from './Tag.root';
+
+const Tag = ({ type, size = DEFAULT_TEXT_SIZE, text, ...rest }: PTag): JSX.Element => {
+    const onClick = (): void => {
```

Comment:
i suppose this was for testing purposes. we should provide a `onClick` prop though